d4tools/test/stat/mean-multiple-ranges/output.txt

```diff
@@ -0,0 +1,2 @@
+chr	0	10	0.9
```

Comment:
while we're here, can you also add, so we have the more general case of something like:
```
chr 0 10 0.9
chr 0 10 0.5
chr 0  8   0.8
chr 1 6 0.6
chr 4 5 2
```
to make sure everything is handled properly?